packages/lab/src/table/Table.css

```diff
@@ -199,4 +203,9 @@ table.saltTable-divider-none tbody tr {
 table.saltTable td {
   height: 100%;
   padding: var(--salt-spacing-100);
+  text-align: start;
```

Comment:
is it worth putting this/th equiv in `table.saltTable td.saltTable-td-align-left` otherwise that class is added but redundant?

Reply:
I don't feel strongly about it, moving it to a new class increases the amount of CSS for the component (only by a small amount). 

There are also other places in the code where we haven't scoped our styling to another class.

I don't think it's redundant though, the class can be used later or used if we support component overrides.